buildpacks/nodejs-function-invoker/src/main.rs

```diff
@@ -60,27 +63,44 @@ impl Buildpack for NodeJsInvokerBuildpack {
     fn build(&self, context: BuildContext<Self>) -> libcnb::Result<BuildResult, Self::Error> {
         log_header("Heroku Node.js Function Invoker Buildpack");
 
+        let app_dir = &context.app_dir;
+        let metadata_runtime = &context.buildpack_descriptor.metadata.runtime;
+        let package_name = &metadata_runtime.package_name;
+        let package_version = &metadata_runtime.package_version;
+
         log_info("Checking for function file");
-        get_main(&context.app_dir).map_err(NodeJsInvokerBuildpackError::MainFunctionError)?;
-
-        context.handle_layer(
-            layer_name!("runtime"),
-            RuntimeLayer {
-                package: context
-                    .buildpack_descriptor
-                    .metadata
-                    .runtime
-                    .package
-                    .clone(),
-            },
-        )?;
+        get_main(app_dir).map_err(NodeJsInvokerBuildpackError::MainFunctionError)?;
+
+        let declared_runtime_package = get_declared_runtime_package(app_dir, package_name)
+            .map_err(NodeJsInvokerBuildpackError::ExplicitRuntimeDependencyFunctionError)?;
+
+        if let Some((package_name, package_version)) = declared_runtime_package {
```

Comment:
`package_name` is being shadowed here. But I think the outer and inner scopes will both have the same contents: `"@heroku/sf-fx-runtime-nodejs"`. I wonder if `get_declared_runtime_package` needs to return the name at all? Maybe it could be `get_runtime_package_version` or similar? We already know the package name, so I think we could remove one memory small memory allocation and a shadowed variable that way.